include/git2/cherrypick.h

```diff
@@ -29,8 +29,8 @@ typedef struct {
 	/** For merge commits, the "mainline" is treated as the parent. */
 	unsigned int mainline;
 
-	git_merge_options merge_opts; /*< Options for the merging */
```

Comment:
I think the arrows are needed for libgit2's docs generator. See (here)[https://github.com/libgit2/docurium]